tests/functional/monitoring/prometheus/metrics/test_ocs_exporter_deployment.py

```
@@ -0,0 +1,337 @@
+    assert (
+        ready_condition.get("status") == "True"
+    ), f"Pod is not ready. Ready condition: {ready_condition}"
+
+    logger.info("✓ Pod readiness status is True (reflects /readyz health)")
+
+    # Verify HTTPS port configuration (RHSTOR-7964 uses port 8443)
+    try:
+        ome_helpers.assert_exporter_uses_https_port(pod_obj)
+        logger.info("✓ Exporter is configured to use HTTPS port 8443")
+    except AssertionError as e:
+        logger.warning(f"HTTPS port verification: {e}")
```

Ruff (0.15.15)

[warning] Logging statement uses f-string

(G004)